packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -113,8 +113,9 @@ module.exports = {
       //    enabled: false,
       //    runs: 200
       //  },
-      //  evmVersion: "byzantium"
-      // }
+      //  evmVersion: "byzantium",
+      // },
+      // disableDefaults: false, // Disable the default solc optimizations
```

Comment:
This comment should explain the nuance of this setting, or why a user may need it.